core/src/services/azblob/backend.rs

```diff
@@ -143,6 +150,45 @@ impl AzblobBuilder {
         self
     }
 
+    /// Set server_side_encryption_customer_key of this backend.
```

Comment:
Please add a comment for what's the content should be. And give an example may be better!

Reply:
Sorry for my recklessness. I should have added comments based on S3 backend. They have now been added.